### ISSUE TITLE
Message formatting problems

### ISSUE THREAD

**Issue by dirk-zimoch** [epics-base/ci-scripts#11]
1. Several activities displayed in yellow have no line feed before the details in gray
```
Loading settingsLoading setup file .ci/defaults.set
Setting up EPICS build systemEPICS_BASE=/home/travis/.cache/base-3.14
Build missing/outdated dependencies\033[32;1mAll dependency modules are up-to-date (nothing to do)\033[0m
```
2. Several messages show raw escape codes instead of colors
```
\033[32;1mEPICS build system already set up (Base was loaded from cache)\033[0m
\033[32;1mAll dependency modules are up-to-date (nothing to do)\033[0m
\033[34;1mDependency module information\033[0m
\033[34;1mContents of RELEASE.local\033[0m
```
You may be missing the -e option to echo.

**Comment by dirk-zimoch** [comment]
I am creating a pull request...

**Comment by dirk-zimoch** [comment]
I made a fork of the .ci repo to test my changes but now it does not build any more in Travis:
```
$ git submodule update --init --recursive
Submodule '.ci' (https://github.com/epics-base/ci-scripts) registered for path '.ci'
Cloning into '/home/travis/build/dirk-zimoch/StreamDevice/.ci'...
error: Server does not allow request for unadvertised object 64e382b46e1d789776f97f94fe2653e76c04fd0c
Fetched in submodule path '.ci', but it did not contain 64e382b46e1d789776f97f94fe2653e76c04fd0c. Direct fetching of that commit failed.
```
https://travis-ci.org/dirk-zimoch/StreamDevice/builds/624067888
Any idea what I am missing?

**Comment by mdavidsaver** [comment]
> I made a fork of the .ci repo to test my changes

Have you updated `.gitmodules` to point to your fork?

**Comment by anjohnson** [comment]
You probably have to change the submodule URL (found in the .gitmodules file, easier to edit that than fiddling around with `git submodule`) in your fork of StreamDevice where you're testing this PR to explicitly point to your ci-scripts fork.

**Comment by mdavidsaver** [comment]
eg https://github.com/mdavidsaver/pvxs/blob/master/.gitmodules

**Comment by dirk-zimoch** [comment]
Right, I still had https://github.com/epics-base/ci-scripts in my .gitmodules.
I changed it now to https://github.com/dirk-zimoch/ci-scripts and it works.
Thanks a lot!

**Comment by dirk-zimoch** [comment]
I can confirm that merge request #12 fixes the formatting.
See Travis build https://travis-ci.org/dirk-zimoch/StreamDevice/builds/624507108

**Comment by ralphlange** [comment]
Fixed by merging #12
